src/sysctl.cr

```diff
@@ -0,0 +1,87 @@
+lib LibC
```

Comment:
Anyway, the variants of sysctl (`sysctl_by_name`) I'm using isn't really POSIX compliant. Plus this call is marked as deprecated on Linux, so how would that play with this generator ?